pkg/apis/events/v1alpha1/cloudstoragesource_validation.go

```diff
@@ -72,6 +80,18 @@ func validateSecret(secret *corev1.SecretKeySelector) *apis.FieldError {
 	return errs
 }
 
+func validateGCPServiceAccount(gServiceAccountName *string) *apis.FieldError {
```

Comment:
can you add a TODO to move this method to a common place, because other Sources are using it and it's in the storage validation?
Event better, maybe you can just move it to apis/duck/v1alpha1/google_cloud_secret_defaults.go... You can rename the file to be something like credentials.go maybe... Then I'll make a pass and move more things around...

Reply:
Sure, I'll move it.